Terminal/user.css

```
@@ -0,0 +1,136 @@
+/*
+Courier New - oldschool
+*/
+* {
+	font-family: 'Courier New';
```

Stylelint (17.4.0)

[error] Unexpected missing generic font family (font-family-no-missing-generic-family-keyword)

(font-family-no-missing-generic-family-keyword)